hnn_core/gui/gui.py

```diff
@@ -1506,8 +1259,12 @@ def run_button_clicked(widget_simulation_name, log_out, drive_widgets,
             simulation_status_bar.value = simulation_status_contents[
                 'finished']
 
-    # Update all plotting panels
-    _initialize_viz_window(simulation_data, analysis_config)
+    viz_manager.add_figure()
+    fig_name = _idx2figname(viz_manager.data['fig_idx']['idx'] - 1)
+    ax_plots = [("ax0", "input histogram"), ("ax1", "current dipole")]
```

Comment:
When you select the axes on the visualization menu, they seem to be flipped from what I would expect (the "ax0" tab controls plots on the bottom of the figure)

Reply:
I don't get it. Do you mean that now ax0 controls the bottom plots?

Reply:
Yeah that's exactly right. At least on my machine when I select ax1 on the GUI, it actually changes `ax[0]` and vice-versa

Reply:
How many figures did you have then

Reply:
I just reproduced the issue.

Reply:
Could you provide a short video clip for that bug?

Reply:
I can't reproduce this issue either.

Reply:
Did a clean install on a different computer and couldn't reproduce. However it seems to persist on my laptop. Going to resolve this for now since it seems to be something messed up with my specific environment